### PR TITLE
added database table sql file

### PR DESCRIPTION
**Pull request by JasperNakamura**
please remove what you guys think we don't need and we also need to add whatever Mohammad states we need to for our reservation but I have no response from him yet.

### CODE REVIEW

File: 291_CarReservation_Table.sql [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+) ON [PRIMARY]
+
+GO
+
+CREATE TABLE [dbo].[Vehicle](
+	[VID] [nchar](4) NOT NULL,
+	[VIN] [nchar](17) NOT NULL,
+	[Make] [nchar](20) NOT NULL,
+	[Model] [nchar](20) NOT NULL,
+	[Color] [nchar](20) NOT NULL,
```
Comment (Taylorr82) [inline 291_CarReservation_Table.sql:84]: perhaps a year on the vehicle?

**Comment by JasperNakamura** [comment]
year added

File: 291_CarReservation_Table.sql [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+	[Province] [nchar](20) NOT NULL,
+	[City] [nchar](50) NOT NULL,
+	[Street] [nchar](100) NOT NULL,
+	[Email] [nchar](60) NOT NULL,
+	[PhoneNum] [int] NOT NULL,
+	[Name] [nchar](50) NOT NULL,
+	[FName] [nchar](50) NOT NULL,
+	[Age] [int] NOT NULL,
+	[Username] [nchar](20) NOT NULL,
+	[Password] [nchar](20) NOT NULL,
```
[the remark this answers is not in the document]
Comment (JasperNakamura) [reply 291_CarReservation_Table.sql:35]: I'm sure hashed passwords make more sense and is more secure but considering this is a introductory course to databases maybe we should leave it as is for now just for ease of making our application

[the remark this answers is not in the document]
Comment (Taylorr82) [reply 291_CarReservation_Table.sql:35]: We should just throw a SHA-256 at it, leaving it plaintext would nullify my degree. I'm sure there's a .NET library with SHA256

**Pull request merged**